investigator/investigator/package_release/investigate_package_release.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+# thoth-investigator
+# Copyright(C) 2020 Bissenbay Dauletbayev
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""This file contains methods used by Thoth investigator to investigate on package releases."""
+
+from thoth.storages.graph import GraphDatabase
+from thoth.messaging import MessageBase
+from thoth.messaging import PackageReleaseMessage
+from thoth.common import OpenShift
+
+from ..metrics import scheduled_workflows
+from .. import common
+from .metrics_package_release import package_release_exceptions
+from .metrics_package_release import package_release_in_progress
+from .metrics_package_release import package_release_success
+
+
+@package_release_exceptions.count_exceptions()
+@package_release_in_progress.track_inprogress()
+def parse_package_release_message(package_release: MessageBase, openshift: OpenShift, graph: GraphDatabase) -> None:
```

Comment:
let's be consistent with the message `package_released` prefix/suffix and in variables should be used everywhere instead of `package_release` :)

Reply:
thanks, https://github.com/thoth-station/investigator/pull/206/commits/ad0030afdbb34257b4211c4798cd3581d752716c